pkg/dao/interfaces.go

```diff
@@ -29,7 +30,7 @@ func GetDaoRegistry(db *gorm.DB) *DaoRegistry {
 		Metrics:    metricsDaoImpl{db: db},
 		Snapshot:   snapshotDaoImpl{db: db},
 		TaskInfo:   taskInfoDaoImpl{db: db},
-		AdminTask:  adminTaskInfoDaoImpl{db: db, pulpClient: pulp_client.GetPulpClient()},
+		AdminTask:  adminTaskInfoDaoImpl{db: db, pulpClient: pulp_client.GetPulpClient(context.Background())},
```

Comment:
Still thinking about the right way to update the pulp client in the request such that it's context has the request ID.

Reply:
But at the moment this would not log anything anyway

Reply:
This might be a similar 'problem', but right now if i turn up db logging (set logging_level to trace), i don't see a request_id on those logs.  This would similarly change as part of each request

Reply:
And this could be helpful, if a certain query takes too long it is logged automatically.  Having the request_id associated with that would be helpful to trace it back to the original request.